tools/Utilities/utils.ps1

```diff
@@ -0,0 +1,56 @@
+﻿# Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+
+<#
+       Install Required Tools
+#>
+Function Install-AzModule {
+    if (!(Get-Module -Name Az -ListAvailable)) {
+        Install-Module -Name Az -Repository PSGallery -Scope CurrentUser -Force -SkipPublisherCheck -AllowClobber 
+        Import-Module Az
+    }
+}
+<#
+    Connect to the Graph Tenant
+#>
+Function Connect-GraphTenant
+{
+    $defaultCertificate = Get-Certificate
+    #Connect To Microsoft Graph Raptor Default Tenant Using ClientId, TenantId and Certificate
+    Connect-MgGraph -Certificate $defaultCertificate -ClientId ${env:CLIENTIDENTIFIER} -TenantId ${env:TENANTIDENTIFIER} | Out-Null
+}
+
+<#
+    Connect to Azure Tenant to Access KeyVault and other resources.
+#>
+Function Connect-AzureTenant
+{
+    $AzureTenantID = ${env:AZURETENANTID}
+    $AzureApplicationID = ${env:AZUREAPPLICATIONID}
+
+    $securePassword = ConvertTo-SecureString -String $env:AZUREAPPCLIENTSECRET -AsPlainText -Force
+
+    $Credential = New-Object -TypeName PSCredential -ArgumentList $AzureApplicationID, $securePassword
+    Connect-AzAccount -Credential $Credential -Tenant $AzureTenantID -ServicePrincipal | Out-Null
+}
+
+<#
+    Get Certificate from Azure KeyVault
+#>
+$global:DefaultCertificate = $null
+Function Get-Certificate
+{
+    if ($null -eq $global:DefaultCertificate)
+    {
+        Connect-AzureTenant
+        # Certificate must be downloaded as a Secret instead of a Certificate to bring down the PrivateKey as well.
+        $keyVaultCertSecret = Get-AzKeyVaultSecret -VaultName ${env:AZUREKEYVAULTNAME} -Name ${env:CERTIFICATENAME}
```

Comment:
We should document that a certificate must be present in Azure KeyVault before executing these tests in our readme/wiki. 

Ideally, we should have a "how-to build and test" section in a wiki/readme that explains all the prerequisites needed to generate, build, and test modules. See https://github.com/microsoftgraph/msgraph-sdk-powershell/wiki/Generation-Process#introduction.

Reply:
Removed references to keyVault as they aren't necessary.

Reply:
I have added a simplified process to the docs.